articles/cms/wordpress/extending.md

```diff
@@ -209,17 +248,124 @@ CSS is retrieved from the wp-admin settings page.
 
 ```php
 /**
- * @param string $css, initialized as empty
+ * Add CSS to the Auth0 login form.
+ *
+ * @param string $css - initialized as empty.
  *
  * @return string
  */
-function a0_docs_ex_auth0_login_css( $css ) {
-	// ... modify $css by replacing or concatenating
+function auth0_theme_hook_auth0_login_css( $css ) {
+	$css .= '
+		body {background: radial-gradient(#01B48F, #16214D)} 
+		#login h1 {display: none}
+		.login form.auth0-lock-widget {box-shadow: none}
+	';
 	return $css;
 }
-add_filter( 'auth0_login_css', 'a0_docs_ex_auth0_login_css' );
+add_filter( 'auth0_login_css', 'auth0_theme_hook_auth0_login_css' );
+```
+
+### auth0_login_form_tpl
+
+Filters the template used for the Auth0 login form. This should return a path to a file containing HTML that replaces what is in `wp-content/plugins/auth0/templates/auth0-login-form.php`. The standard Lock initiation JS looks for an ID attribute of `auth0-login-form` to instantiate the login form so make sure that's present or replace the `wp-content/plugins/auth0/assets/js/lock-init.js` file with your own. 
+
+```
+/**
+ * Override the Lock login form template.
+ *
+ * @param string  $tpl_path - original template path.
+ * @param array   $lock_options - Lock options.
+ * @param boolean $show_legacy_login - Should the template include a link to the standard WP login?
+ *
+ * @return string
+ */
+function auth0_theme_hook_auth0_login_form_tpl( $tpl_path, $lock_options, $show_legacy_login ) {
+	return AUTH0_THEME_ROOT . '/templates/auth0-login-form.html';
+}
+add_filter( 'auth0_login_form_tpl', 'auth0_theme_hook_auth0_login_form_tpl', 10, 3 );
+```
+
+### auth0_settings_fields
+
+This filter is used to modify an existing form field or to add a new one. This should return a modified `$options` array with your changes or additions. New fields must have a field callback, as shown below. 
+
+```
```

Comment:
set the scripting lang to ensure proper formatting

Reply:
☝️